lte/gateway/c/oai/tasks/sgw/sgw_handlers.c

```diff
@@ -685,145 +726,176 @@ int sgw_handle_sgi_endpoint_deleted(
   OAILOG_FUNC_RETURN(LOG_SPGW_APP, rv);
 }
 
+//------------------------------------------------------------------------------
+static void populate_sgi_end_point_update(
+    sgw_eps_bearer_ctxt_t* eps_bearer_ctxt_p,
+    const itti_s11_modify_bearer_request_t* const modify_bearer_pP,
+    itti_sgi_update_end_point_response_t* sgi_update_end_point_resp,
+    uint8_t sgi_rsp_idx, uint8_t* idx, imsi64_t imsi64) {
```

Comment:
is there a reason why idx is passed as a pointer and not passed by value as uint8_t?